src/StudentProfileProjectSchema.js

```diff
@@ -0,0 +1,131 @@
+//
+// # StudentProfileProjectSchema
+//
+// This schema describes de "projects" shown in student's profiles in the job
+// placement app. These should not be confused with documents in the `projects`
+// collection (`ProjectSchema`), which describe projects as used in our training
+// programs (ie: Bootcamp).
+//
+// Data in this collection is manages via two different "pages":
+//
+// 1. Students that have been assigned to a "job placement" cohort will be able
+//    to manage the projects shown in their profile from the main "Settings"
+//    page.
+// 2. Cohort admins can also manage each user's projects via the "cohort users"
+//    admin page.
+//
+// Indexes:
+//
+// * `student`: Basic index to enhance filtering by student. Note that this
+//   index is NOT _unique_, which means that a given student can have many
+//   `StudentProfileProject`s.
+// * `repo`: A _unique_ index to ensure that the same repo can not be used more
+//   than once.
+//
+// Migration notes:
+//
+// * `student`: Previously the _key_ in the main `user` object.
+// * `repo`: Previously known as `github`.
+// * `url`: Previously known as `demo`.
+// * `name`: Stays the same.
+// * `description`: Stays the same.
+// * `tags`: Used to be a list of comma separated strings and is now an array of
+//   strings.
+// * `image`: Stays the same.
+// * `where`: Stays the same.
+// * `date`: Stays the same.
+//
+// The following fields have been ignored from object in `githubProjects` array:
+//
+// * `id`: Number
+// * `repoErrors`: ??
+// * `error`: ??
+// * `url`: ??
+// * `license`: ??
+//
+// Example old githubProjects in Firestore:
+//
+// "githubProjects": {
+//   "0": {
+//     "github": "https://github.com/lupomontero/media",
+//     "demo": "https://lupomontero.github.io/media/",
+//     "name": "Foo",
+//     "description": "La descripción del proyecto...",
+//     "tags": "foo,bar,baz",
+//     "image": "http://foo.bar/baz.png",
+//     "where": "Laboratoria",
+//     "date": "2018-11",
+//     "id": 0
+//   }
+// }
+//
+// Example new student profile projects:
+//
+// [
+//   {
+//     "student": "ObjectId",
+//     "repo": "https://github.com/lupomontero/media",
+//     "url": "https://lupomontero.github.io/media/",
+//     "name": "Foo",
+//     "description": "La descripción del proyecto...",
+//     "tags": ["foo","bar","baz"],
+//     "image": "http://foo.bar/baz.png",
+//     "where": "Laboratoria",
+//     "date": "2018-11"
+//   }
+// ]
+//
+
+module.exports = (conn) => {
+  const StudentProfileProjectSchema = new conn.Schema({
+    student: {
```

Comment:
If we change the schema name to `JobPlacementProfileEndorsementSchema`, would it make sense to rename this _field_ to simply `user`?

cc/ @gmoura